onnxruntime/core/providers/cuda/math/matmul.h

```diff
@@ -28,6 +32,7 @@
   const bool trans_B_;
   const bool trans_batch_a_;
   const bool trans_batch_b_;
+  const bool disable_cublaslt_matmul_;
```

Comment:
>disable_cublaslt_matmul_; [](http://example.com/codeflow?start=13&length=25)

It is redundant to have sperate Boolean for each operator

Reply:
See my comment below (have a case for not doing the centralized boolean approach)